fe/fe-core/src/main/java/com/starrocks/load/InsertOverwriteJob.java

```diff
@@ -45,27 +45,32 @@ public class InsertOverwriteJob {
     @SerializedName(value = "sourcePartitionNames")
     private List<String> sourcePartitionNames;
 
+    @SerializedName(value = "dynamicOverwrite")
+    private boolean dynamicOverwrite = false;
 
     private transient InsertStmt insertStmt;
 
```

Comment:
Better to add an empty constructor. 
The default value will be null after deserialization without empty constructor.